compiler/src/dmd/typesem.d

```diff
@@ -6229,6 +6229,17 @@ Type referenceTo(Type type)
     return type.rto;
 }
 
+// Make corresponding static array type without semantic
+Type sarrayOf(Type type, dinteger_t dim)
+{
+    assert(type.deco);
+    Type t = new TypeSArray(type, new IntegerExp(Loc.initial, dim, Type.tsize_t));
+    // according to TypeSArray::semantic()
```

Comment:
might as well update the comment to use `.`, instead of `::`. also is semantic still a member function of `TypeSArray`?